chainer/training/extension.py

```diff
@@ -28,14 +28,10 @@ class Extension(object):
             ``(1, 'iteration')`` by default.
         priority: Default priority of the extension. It is set to
             ``PRIORITY_READER`` by default.
-        invoke_before_training: Default flag to decide whether this extension
```

Comment:
I found that adding a property will let IDEs suggest the property which is misleading. I will write `__getattr__` that raises a special error when the name is `invoke_before_training`.